src/rendezvous/channels/MSC3903ECDHv2RendezvousChannel.ts

```diff
@@ -98,6 +99,7 @@ export class MSC3903ECDHv2RendezvousChannel<T> implements RendezvousChannel<T> {
                 key: encodeUnpaddedBase64(this.ourPublicKey),
                 transport: await this.transport.details(),
             },
+            flow,
             intent,
```

Comment:
While trying to get a handle on how these pieces fit together, I find it a bit odd that this is implementing a "MSC3903 rendezvous channel", but there is no mention in MSC3903 of this bit. If this is specific to MSC3906, shouldn't it be in `MSC3906Rendezvous` rather than here?